airqo-platform/AirQo-api
# Email verification after user registration:

- What does this PR do?
This PR introduces new endpoints for email verification under the user authentication service. 

- What JIRA task is related to this PR?
No link available

- How do I test out this PR?
Git clone this repo `cd AirQo-api/src/auth-service`
Create a docker image locally: `docker build -t us.gcr.io/airqo-250220/airqo-auth-service-api .`
Afterwards, run the newly created image: `docker run -d -p 5000:3000 us.gcr.io/airqo-250220/airqo-auth-service-api`
Using your REST client, test out the endpoints `confirm` and` /resend` for the`auth-service`, use port 5000 as specified above or change to what you would like to use.

**Alternatively.......**
 `cd AirQo-api/src/auth-service`
`npm install`

For MAC users....
`npm run prod-mac` for "production"
`npm run dev-mac` for "development"

For Windows users
`npm run prod-pc` for  "production"
`npm run dev-pc` for  "development"


- Which end points should are ready for testing?
`verify email` , and `resend Token`,
inside the `auth-service` sheet of the [API documentation](https://docs.google.com/spreadsheets/d/1-NGTDBtTcIi8Fg2UYSWjcK6geKrZdopz5kxRFsCEMPs/edit#gid=897606332).

- **OPTIONAL for testing:** 
One could also push to the Google Container Registry using `docker push us.gcr.io/airqo-250220/airqo-auth-service-api`
